src/static/containers/ZeroCard.jsx

```diff
@@ -216,16 +216,18 @@ ZeroCard.propTypes = {
         waste_reduction_score: PropTypes.number.isRequired,
         difficulty_score: PropTypes.number.isRequired,
         cost_score: PropTypes.number.isRequired,
-        help_links: PropTypes.string,
+        help_links: PropTypes.arrayOf(PropTypes.string),
         card_stats: PropTypes.shape({
-            co2_reduction: PropTypes.number,
-            waste_reduction: PropTypes.number,
-            water_use_reduction: PropTypes.number,
-            data_sources: PropTypes.shape({
-                name: PropTypes.string,
-                link: PropTypes.string,
-                status: PropTypes.string,
-            }),
+            co2_reduction: PropTypes.string,
```

Comment:
Why string? We should send numbers here.

Reply:
https://github.com/encode/django-rest-framework/issues/508

Changed the settings of REST_FRAMEWORK not to send string instead of decimal